exercises/concept/health-statistics/tests/health-statistics.rs

```diff
@@ -5,6 +5,7 @@ const AGE: u32 = 89;
 const WEIGHT: f32 = 131.6;
 
 #[test]
+#[ignore]
```

Comment:
```suggestion

```
We don't want to ignore the first test, I believe.

Reply:
Thanks Bob! The first test is now active.

Note to self: From the [document](https://github.com/exercism/legacy-docs/blob/main/language-tracks/exercises/anatomy/test-suites.md) linked in the [exercise tests](https://github.com/exercism/rust#exercise-tests) section

> In order to provide a better user experience, some language tracks only leave the first test in an exercise test suite active. All subsequent tests are marked as "skipped" or "pending" using whatever method or directive that is provided by the testing framework. This means that the person solving the exercise must manually edit the test suite to activate the tests one by one as they write their solution.